extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -766,7 +766,7 @@ export class AmpA4A extends AMP.BaseElement {
           if (this.experimentalNonAmpCreativeRenderMethod_ ==
               XORIGIN_MODE.NAMEFRAME) {
             this.preconnect.preload(
-                getDefaultBootstrapBaseUrl(this.win, 'nameframe'));
+                getDefaultBootstrapBaseUrl(this.win, 'nameframe'), 'document');
```

Comment:
/cc @ampproject/a4a. I believe you deoptimized Safari from the nameframe experiment because it wouldn't preload the document and reuse it in the fetch. Would that issue be fixed by this?

Reply:
We explicitly return nameframe for this header when used so I'm not sure I follow